concordium-std/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased changes
 
+- Set minimum Rust version to 1.67.
```

Comment:
I think this is no longer true for concordium-std. This should go to the testing library's changelog.

Reply:
Removed the change log from `concordium-std`.

The testing library's changelog has already the following statement:
```
- Bump minimum supported Rust version to `1.67`.
```